vue/src/components/loki/decodeQuery.ts

```diff
@@ -0,0 +1,109 @@
+// decode query from url and transform into actual query
+
+// updates labels from query
+
+export function decodeQuery(query: string, apiUrl: string, labels = []): any {
```

Comment:
Any chance we can add a type instead of `any`?

Reply:
decodeQuery helper wasn't used. Removing file.